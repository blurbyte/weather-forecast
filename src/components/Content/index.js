import styled from 'styled-components';

const Content = styled.div`
  max-width: 100rem;
  padding: 0 2rem;
  margin: 0 auto;
`;

export default Content;
