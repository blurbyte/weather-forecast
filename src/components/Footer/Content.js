import styled from 'styled-components';

import BaseContent from '../Content';

const Content = styled(BaseContent)`
  display: flex;
  align-items: center;
  height: 100%;
`;

export default Content;
