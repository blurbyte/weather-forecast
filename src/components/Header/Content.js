import styled from 'styled-components';

import BaseContent from '../Content';

const Content = styled(BaseContent)`
  display: flex;
  align-items: center;
`;

export default Content;
