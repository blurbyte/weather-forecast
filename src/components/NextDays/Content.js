import BaseContent from '../Content';

const Content = BaseContent.withComponent('section');

export default Content;
