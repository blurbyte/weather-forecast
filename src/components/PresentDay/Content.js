import styled from 'styled-components';

import media from '../../styles/media';
import BaseContent from '../Content';

const Section = BaseContent.withComponent('section');

const Content = styled(Section)`
  display: flex;
  flex-direction: column;
  align-items: center;
  padding: 4rem 2rem;

  ${media.phone`
    padding-top: 2rem;
    padding-bottom: 2rem;
  `};
`;

export default Content;
