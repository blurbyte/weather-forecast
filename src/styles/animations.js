import { keyframes } from 'styled-components';

export const fadeIn = keyframes`
  from {
    opacity: 0;
    transform: translateY(2rem);
  }
  to {
    opacity: 1;
    transform: translateY(0);
  }
`;
