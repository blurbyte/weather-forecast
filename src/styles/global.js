import { injectGlobal } from 'styled-components';

injectGlobal`
  @import url('https://fonts.googleapis.com/css?family=Cardo:700&subset=latin-ext');

  html {
    font-size: 62.5%;
  }

  body {
    background: #71d7ed;
    color: #222222;
    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif;
    line-height: 1.4;
    font-size: 1.4rem;
    font-weight: 400;
    overflow-y: scroll;
    user-select: none;
  }
`;
