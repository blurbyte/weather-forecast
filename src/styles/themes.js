export const dayTheme = {
  primary: '#222222',
  background: '#71d7ed'
};

export const nightTheme = {
  primary: '#ffffff',
  background: '#3033ba'
};
