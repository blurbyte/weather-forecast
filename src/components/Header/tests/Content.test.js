import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { dayTheme } from '../../../styles/themes';
import Content from '../Content';

test('renders correctly', () => {
  const component = renderer.create(<Content theme={dayTheme} />);
  expect(component.toJSON()).toMatchSnapshot();
});
