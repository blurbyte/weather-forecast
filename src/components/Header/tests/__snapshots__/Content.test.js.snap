// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c1 {
  height: 100%;
  max-width: 96rem;
  margin: 0 auto;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0 c1"
/>
`;
